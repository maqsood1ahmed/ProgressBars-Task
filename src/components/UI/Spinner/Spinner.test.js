import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import LoadingSpinner from './Spinner';

configure({ adapter: new Adapter() });

describe('Loading Spinner test', () => {
    it('should be defined', () => {
        expect(LoadingSpinner).toBeDefined();
    });
    it('should render correctly', () => {
        const tree = shallow(<LoadingSpinner />);
        expect(tree).toMatchSnapshot();
    });
});
