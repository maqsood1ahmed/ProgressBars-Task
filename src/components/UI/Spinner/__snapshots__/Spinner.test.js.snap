// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Spinner test should render correctly 1`] = `ShallowWrapper {}`;
